sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -379,3 +391,77 @@ export function readableSpanToEnvelope(span: ReadableSpan, ikey: string): Envelo
     },
   };
 }
+
+/**
+ * Span Events to Azure envelopes parsing.
+ * @internal
+ */
+export function spanEventsToEnvelopes(span: ReadableSpan, ikey: string): Envelope[] {
+  let envelopes: Envelope[] = [];
+  if (span.events) {
+    span.events.forEach((event: TimedEvent) => {
+      let baseType: "ExceptionData" | "MessageData";
+      const sampleRate = 100;
+      let time = new Date(hrTimeToMilliseconds(event.time));
+      let name = "";
+      let baseData: TelemetryExceptionData | MessageData;
+      const properties = createPropertiesFromSpanAttributes(event.attributes);
+      const tags: Tags = createGenericTagsFromSpan(span);
+
+      if (event.name == "exception") {
+        name = "Microsoft.ApplicationInsights.Exception";
+        baseType = "ExceptionData";
+        let typeName = "";
+        let message = "Exception";
+        let stack = "";
+        let hasFullStack = false;
+        if (event.attributes) {
+          typeName = String(event.attributes[SemanticAttributes.EXCEPTION_TYPE]);
+          stack = String(event.attributes[SemanticAttributes.EXCEPTION_STACKTRACE]);
+          if (stack) {
+            hasFullStack = true;
+          }
```

Comment:
Does ot js sdk populated [escaped](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/exceptions.md#attributes) ?

Reply:
Maybe, where it ends in Azure Monitor? properties?